DataBase.cpp

```diff
@@ -1 +1,24 @@
 #include "DataBase.hpp"
+
+#include <iostream>
+#include <string>
+#include <iomanip>
+#include <fstream>
+
+using namespace std;
+
+//overloading of << operator 
+ostream& operator << (ostream& stream_out, Person p) 
+{  
```

Comment:
formatowanie: we wszystkich innych funckjach nawias wasaty stawialismy w tej samej lini do deklaracja fukncji